canonical-websites/www.ubuntu.com
# Remove autoprefixer and add inline prefixes

Done
Remove autoprefixer and add inline prefixes. Simplify the npm scripts.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the site still looks the same
